### PR TITLE
19222 - FAS - update view to include cheque advise details

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:* /bcgov/entity[19222](https://github.com/bcgov/entity/issues/19222)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/ViewRoutingSlip/RefundRequestForm.vue [hunk 1]
```diff
@@ -26,21 +26,25 @@
         >
         </address-form>
       </v-col>
-      <v-col class="col-3 font-weight-bold"
-        :class="canEdit ? 'pt-0' : ''">
+      <v-col
+        class="col-3 font-weight-bold"
+        :class="canEdit ? 'pt-0' : ''"
+        v-if="isEditing || (!isEditing && chequeAdvice)"
```
Comment (Jxio) [inline src/components/ViewRoutingSlip/RefundRequestForm.vue:32]: Hide chequeAdvice field when it's empty

File: src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts [hunk 2]
```diff
@@ -77,9 +77,9 @@ export default function useRoutingSlipInfo (props) {
 
   const showAddress = computed(() => {
     return (
-      (isRefundProcess(currentStatus?.value) && canRequestRefund.value) ||
-        false
-    )
+      isRefundProcess(currentStatus?.value) ||
+      currentStatus?.value?.code === SlipStatus.ACTIVE
```
Comment (seeker25) [inline src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts:81]: It already shows when it's in `IsRefundProcess` which is ACTIVE

File: src/components/ViewRoutingSlip/RefundRequestForm.vue (hunk 1 above)
Comment (seeker25) [reply src/components/ViewRoutingSlip/RefundRequestForm.vue:32]: `isEditing || (!isEditing && chequeAdvice)` -> `isEditing || chequeAdvice`

File: src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts (hunk 2 above)
Comment (seeker25) [reply src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts:81]: IC `Hide chequeAdvice field when it's empty` ok should be good

Comment (Jxio) [reply src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts:81]: image
image

I looked at the IsRefundProcess function, ACTIVE not in, IsRefundProcess referenced on few other spots, so I think better not add ACTIVE to it.

Comment (seeker25) [reply src/composables/ViewRoutingSlip/useRoutingSlipInfo.ts:81]: Yup, should be fine, I didn't see the condition you added above

**Review by seeker25 — APPROVED** [review]

**Pull request merged**